pixeltable/exec/sql_scan_node.py

```diff
@@ -47,8 +53,9 @@ def __init__(
             sql_subexprs = iter_arg.subexprs(filter=lambda e: e.sql_expr() is not None, traverse_matches=False)
             [self.sql_exprs.append(e) for e in sql_subexprs]
         self.filter = filter
-        self.filter_eval_ctx = \
+        self.filter_eval_ctx = (
```

Comment:
I feel like this is an argument for trying out other auto-formatter options. I already volunteered.

Reply:
Again, I prefer this.